hello_world/Makefile

```diff
@@ -0,0 +1,11 @@
+export V?=0
+
```

Comment:
To let this Makefile work with tools that only provide CROSS_COMPILE we could add the following here:

{code}
# If _HOST or _TA specific compilers are not specified, then use CROSS_COMPILE
HOST_CROSS_COMPILE ?= $(CROSS_COMPILE)
TA_CROSS_COMPILE ?= $(CROSS_COMPILE)
{code}

Reply:
added

Reply:
Don't we need to plit this in 2 lines?
```makefile
V ?= 0
export V
```

Reply:
@etienne-lms https://www.gnu.org/software/make/manual/html_node/Variables_002fRecursion.html

```
export variable := value
```
has the same result as:
```
variable := value
export variable
```